### PR TITLE
compleasm: Fix the lineage list problem

### PR DESCRIPTION
**Pull request by rlibouba**
Hello, 
I've opened this PR to correct a problem I noticed once the tool was available on usegalaxy.eu. The list with the different lineages did not appear. I made some modifications (using busco as an example) to correct this.

### CODE REVIEW

File: tools/compleasm/compleasm.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
     -a '$input'
     -o galaxy
     --mode $mode
-    -l '$busco_database'
```
Comment (abretaud) [inline tools/compleasm/compleasm.xml:14]: Hum, lineage_dataset is a subdir of busco_database right?

Comment (rlibouba) [reply tools/compleasm/compleasm.xml:14]: lineage_dataset corresponds to the list of lineages defined in macros.xml. Perhaps the name of the parameter isn't chosen correctly?

**Comment by bgruening** [comment]
ping @abueg 
I guess this is the error you are seeing.

**Review by abretaud — APPROVED** [review]
Nice, should be ok now

**Comment by bgruening** [comment]
Its failing with:

> OSError: [Errno 30] Read-only file system: '/home/runner/work/tools-iuc/tools-iuc/tools/compleasm/test-data/test-db/busco_downloads/lineages/file_versions.tsv.tmp'

I guess we need to copy this file into the JWD?

**Review by abretaud — APPROVED** [review]

**Pull request merged**